Packs/GoogleDrive/Integrations/GoogleDrive/GoogleDrive.yml

```diff
@@ -1807,6 +1824,17 @@ script:
       predefined:
       - "True"
       - "False"
+    - auto: PREDEFINED
+      description: Bodies of items (files/documents) to which the query applies. Prefer 'User' or 'Drive' to 'All Drives' for efficiency. By default, corpora is set to 'User'. However, this can change depending on the filter set through the 'query' argument.
```

Comment:
Why "Bodies of items"?  Can't it just be Items?

Reply:
How about "Files or documents to which the query applies."

Reply:
As discussed with Shirley offline, changing to "Files or documents to which the query applies."